src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  border-radius: 5px;
+  width: 200px;
+  border: 1px solid $gray-elements-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-accent-color;
+    padding-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+    padding-bottom: 16px;
+  }
+
+  &__reviews-line {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+  }
+
+  &__reviews-number {
+    display: flex;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $main-accent-color;
+  }
+
+  &__price-line {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+
+  &__pcice-name {
```

Comment:
There is a typo in the class name, change it from '__pcice-name' to '__price-name'.